packages/wdio-reporter/reporter.d.ts

```diff
@@ -19,11 +23,36 @@ declare namespace WDIOReporter {
         onSuiteEnd(suite: Suite): void;
         onRunnerEnd(runner: any): void;
 
-        isSynchronised: boolean;
+        private _isSynchronised: boolean;
 
         write(content: any): void;
     }
 
+    interface Runner {
```

Comment:
Why do we need this interface in the reporter?

Reply:
we are using Runner interface in sumologic  class, hence i have added this in reporter interface, so that it becomes easy to accessible in index.ts file for sumologic package